api/functions/orm_to_dict.py

```diff
@@ -0,0 +1,10 @@
+def orm_to_dict(sql):
+    d1 = []
```

Comment:
Same idea here.  I really don't understand what `d1` and `d2` are.  I'd assume dict, but again I am a fan of being explicit